packages/form/components/controls/TextArea.astro

```diff
@@ -0,0 +1,41 @@
+---
+/**
+ * TEXT AREA COMPONENT
+ */
+import type { FormControl } from '../../core/form-control';
+
+export interface Props {
+	control: FormControl;
```

Comment:
Then here control should be a TextArea :)

Reply:
Maybe we need a naming convention to distinguish between the astro components and the classes...